main.go

```diff
@@ -1,22 +1,62 @@
 package main
 
 import (
+	"context"
 	"fmt"
 	"log"
 	"net/http"
 	"os"
 
+	"gitlab.com/NebulousLabs/errors"
+
 	"github.com/NebulousLabs/skynet-accounts/api"
+	"github.com/NebulousLabs/skynet-accounts/database"
+)
+
+var (
+	// envDBHost holds the name of the environment variable for DB host.
+	envDBHost = "SKYNET_DB_HOST"
+	// envDBPort holds the name of the environment variable for DB port.
+	envDBPort = "SKYNET_DB_PORT"
+	// envDBUser holds the name of the environment variable for DB username.
+	envDBUser = "SKYNET_DB_USER" // #nosec
```

Comment:
q: Why disable gosec here? 

nit: Can you add the error, e.g. `#nosec GXXX`?

Reply:
A: It was suspecting that I'm hard-coding the credentials to the DB. This is an error that we're supposed to verify and disable if it's a false alarm.

> nit: Can you add the error, e.g. #nosec GXXX?

Good idea!

Reply:
Cool, with the error specified we only disable the one that we explicitly expect 🙂